### DISCUSSION TITLE
Better To Keep Training old model with new data, or train fresh model with old and new data?

### DISCUSSION THREAD

**Discussion by quitmeyer** [ultralytics/ultralytics#16811] [Q&A] [answered]
I have a question about iteratively training models with YOLO

So I have a yolo11s-obb model that i trained with like 500 images only one category, and it works ok. Let's call that MODEL_A

I then did some more ground-truth labeling and have 1000 images and trained another fresh yolo11s-obb model and it works even better. Let;s call it MODEL_B

Now, i have labelled 3000 images (2000 new images plus the 1000 that MODEL_B was trained on).

Is it better to 
1) Train a fresh yolo11s-obb model with all 3000 images

OR

2) train MODEL_B with the 2000 new images ?

or is it

3) both options are effectively the same?

**Comment by UltralyticsAssistant** [comment]
👋 Hello @quitmeyer, thank you for your interest in Ultralytics 🚀!

Your question about iterative training with YOLO is a great one! While we wait for an Ultralytics engineer to provide a detailed response, I recommend checking our [Docs](https://docs.ultralytics.com) for training insights where you might find helpful tips.

If you lean toward any specific approach, it might be beneficial to provide some additional details, such as training logs or specific challenges you’re encountering, which would help in understanding the nuances of your setup.

For iterative training and data strategies, make sure to go through our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

Join our community for diverse insights:
- Engage in real-time on [Discord](https://ultralytics.com/discord) 🎧
- Explore discussions on [Discourse](https://community.ultralytics.com)
- Share and learn with others on our [Subreddit](https://reddit.com/r/ultralytics)

## Upgrade

Ensure you're using the latest `ultralytics` package:

```bash
pip install -U ultralytics
```

## Environments

Explore these verified environments for running your training:
- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/ultralytics/yolov8"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule) tests are passing. They verify correct operation across platforms.

Hope this helps for now! An Ultralytics engineer will assist you shortly. 😊

**Comment by glenn-jocher** [comment, marked as answer]
Training a fresh model with all 3000 images is generally recommended to ensure the model learns from the entire dataset comprehensively.
